source/documentation/breaking-changes/import.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Breaking Change: `@import` and global built-in functions'
+introduction: >
+  Originally, Sass used `@import` rules to load other files with a single
+  global namespace, with all built-in functions also available globally. We're
+  deprecating both Sass `@import` rules and global built-in functions now that
+  the module system (`@use` and `@forward` rules) has been available for
+  several years.
+---
+
+Sass historically loaded dependencies in a single global namespace using
+`@import` rules. This causes numerous problems, requiring Sass members to be
```

Comment:
You don't need to repeat the introduction here; it'll be right above this text.

Reply:
Fixed